packages/atoms/src/Radio.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import {
+    NativeStyleObject,
+    NativeStyleUtils,
+    prepareNativeStyle,
+    useNativeStyles,
+} from '@trezor/styles';
+import { Pressable, View } from 'react-native';
+
+type RadioValue = string | number | undefined;
```

Comment:
Why do you use `string | number | undefined` when passed value can only be string? Shouldn't there be overlap between value passed to the component and value that the onPress accepts?